tests/bluetooth/shell/src/main.c

```diff
@@ -2052,12 +2077,7 @@ static void l2cap_disconnected(struct bt_l2cap_chan *chan)
 	printk("Channel %p disconnected\n", chan);
 }
 
-static struct net_buf *l2cap_alloc_buf(struct bt_l2cap_chan *chan)
-{
-	printk("Channel %p requires buffer\n", chan);
-
-	return net_buf_alloc(&data_rx_pool, K_FOREVER);
-}
+static struct net_buf *l2cap_alloc_buf(struct bt_l2cap_chan *chan);
```

Comment:
I guess we can remove this forward declaration and have l2cap_alloc_buf directly here, or you do have any other problem with that?